### ISSUE TITLE
Workspace with discord icon is reversed

### ISSUE THREAD

**Issue by volucris1** [pierrechevalier83/workstyle#38]
![image](https://user-images.githubusercontent.com/46374280/161431004-45e7a44b-43fc-4ce5-a081-689dba6e6970.png)
My config
```toml
# Config for workstyle
#
# Format:
# "pattern" = "icon"
#
# The pattern will be used to match against the application name, class_id or WM_CLASS.
# The icon will be used to represent that application.
#
# Note if multiple patterns are present in the same application name,
# precedence is given in order of apparition in this file.

"mpv" = ""
"playlofi" = " Lo-Fi"

"whatsapp" = ""
"alacritty" = ""
"github" = ""
"rust" = ""
"private browsing" = ""
"firefox" = ""
"chrome" = ""
"chromium" = ""
"brave" = "WWW"
"nautilus" = ""
"libreoffice calc" = ""
"libreoffice writer" = ""
"libreoffice" = ""
"nvim" = ""
"gthumb" = ""
"menu" = ""
"calculator" = ""
"transmission" = ""
"videostream" = ""
"music" = ""
"disk usage" = ""
"code" = ""
".pdf" = ""
"telegram" = " "
"marktext" = " "
"todoist" = ""
"zathura" = ""
"discord" = "ﭮ"

[other]
fallback_icon = "dunno |"
deduplicate_icons = false
```
And my waybar font is FuraCode Nerd Font

**Comment by MaxVerevkin** [comment]
It can happen with some icons, see https://github.com/greshake/i3status-rust/issues/1206

The easiest solution is to use a different icon.

**Comment by pierrechevalier83** [comment]
Indeed, it happens with some icons. Ultimately, it's due to this issue with nerdfonts: https://github.com/ryanoasis/nerd-fonts/issues/365. Some icons appear like arabic characters and change the direction of writing from left-to-right to right-to-left.

I have experienced this issue too and I've used the workaround of prepending the discord icon with a unicode left-to-right character like so:
```
"discord" = "\u200eﭮ"
```

It's not ideal, but it works around the issue.

I'm hesitant to hard-code a workaround in workstyle itself since this workaround in the config works and things should eventually resolve themselves once nerdfont remaps the icons to properly fix the issue.
